modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/Environment.scala

```diff
@@ -33,7 +33,7 @@ import com.snowplowanalytics.snowplow.rdbloader.config.CliConfig
 /** Container for most of interepreters to be used in Main
  * JDBC will be instantiated only when necessary, and as a `Reousrce`
  */
-class Environment[F[_]](cache: Cache[F], logging: Logging[F], iglu: Iglu[F], aws: AWS[F], val state: State.Ref[F]) {
+class Environment[F[_]](cache: Cache[F], logging: Logging[F], iglu: Iglu[F], aws: AWS[F], val state: State.Ref[F], val blocker: Blocker) {
   implicit val cacheF: Cache[F] = cache
   implicit val loggingF: Logging[F] = logging
   implicit val igluF: Iglu[F] = iglu
```

Comment:
Why do we need to have these `implicit` values ?

Reply:
They're interpreters https://github.com/snowplow/snowplow-rdb-loader/blob/48a00af134bc40595c93f6db60b2fd6f1be65c22/modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/Main.scala#L65